game/definitions.rpy

```diff
@@ -26,6 +26,11 @@ default persistent.jn_player_appearance_hair_length = None
 default persistent.jn_player_appearance_hair_colour = None
 default persistent.jn_player_appearance_height_cm = None
 
+# Nickname data
+default persistent.jn_player_nicknames_allowed = True
+default persistent.jn_player_nicknames_current_nickname = "Natsuki"
+default persistent.jn_player_nicknames_bad_given_total = 0
+
```

Comment:
Can we move these to above the nickname topic? Easier to keep track of

We should do this for all topics that have their own persistent variables tied to them